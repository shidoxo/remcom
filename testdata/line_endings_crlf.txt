// First comment with CRLF
Some content here // Inline comment
// Another comment

More content // More inline
// Final comment