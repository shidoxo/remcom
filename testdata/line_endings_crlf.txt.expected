Some content here

More content